README.md

```diff
@@ -0,0 +1,58 @@
+# Executing ELF binaries
```

Comment:
I made the commit titles looking similar. Since this is the app repository of `elfloader`, I'd prefer a shorter, more general title, like "Readme: Section about debugging". I think this is true because we are also executing and debugging the `elfloader` unikernel (and not only an ELF binary). What do you think?

Reply:
Thank you for the overall updates. :) Indeed, it looks fine to be more general.